docs/advanced-features/custom-error-page.md

```diff
@@ -55,3 +55,23 @@ export default Page
 ```
 
 The `Error` component also takes `title` as a property if you want to pass in a text message along with a `statusCode`.
+
+## Default Static 404 Page
+
+When you don't need to create a custom `_error` page and would prefer to use the default Next.js provided one, we will automatically apply the static optimization to this file during a production build and use it when deploying on [Now](https://zeit.co) or with `next start`.
+
+## Static `pages/404`
+
+For the cases where you do want a custom `_error` page but still want the benefits of a static 404 page we have added a convention of a `pages/404` file that takes priority over `_error` for 404s specifically. This 404 page is specifically meant for creating a static 404 page and strictly does not allow `getInitialProps` or `getServerSideProps` to be used.
```

Comment:
- Maybe add `.js` (even though it could be TS, I think people will understand): `pages/404.js`
- Earlier it said "`pages/_error.js` is only used in production for non-404 errors" but this paragraph says "takes priority over `_error` for 404s" - so 404 may be rendered by `_error`? In that case, the earlier sentence was incorrect?
- `specifically` is used twice, I think the second one can be removed.
- How about `getStaticProps`?

Reply:
Good point, I reworked it using the above mentioned structure. Let me know if this seems better